bucket/dualmonitortaskbar.json

```diff
@@ -0,0 +1,28 @@
+{
+    "homepage": "http://sourceforge.net/projects/dualmonitortb/",
+    "version": "1.22.021813",
+    "license": {
```

Comment:
```suggestion
    "license": "GPL-3.0-or-later",
```

Reply:
Wouldn't this implies removing the `url` link to the licence?

Reply:
It is valid SPDX identifier. There is no need for URL